agent/component-test/softwarecontaineragent_componenttest.cpp

```diff
@@ -105,7 +105,7 @@ class SoftwareContainerAgentTest: public ::testing::Test
 
         try {
             std::shared_ptr<SoftwareContainerFactory> factory = std::shared_ptr<SoftwareContainerFactory> (new SoftwareContainerFactory());
-            std::shared_ptr<ContainerUtilityInterface> utility = std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface());
+            std::shared_ptr<ContainerUtilityInterface> utility = std::shared_ptr<ContainerUtilityInterface> (new ContainerUtilityInterface(config));
```

Comment:
I would prefer you only pass the one value you need from the config. If we need more values later on we can change it to take the whole config as an argument.